gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/GemFireCacheImpl.java

```diff
@@ -857,6 +870,8 @@ public void runOldEntriesCleanerThread(){
   class OldEntriesCleanerThread implements Runnable {
     // Keep each entry alive for at least 20 secs.
     Random r = new Random();
+    MapCallback queueRemover = new QueueRemover();
+    MapCallback entryRemover = new MapCallbackAdapter();
```

Comment:
Why is entryRemover required since it is just the default MapCallbackAdapter? Why not just use the two argument remove() method which should be equivalent?

Reply:
Will use it. was confused to see the condition being passed null there.